### DISCUSSION TITLE
Problem Login Apple Account after update for MACOS Sequoia 15.2

### DISCUSSION THREAD

**Discussion by wperesdev** [sickcodes/Docker-OSX#852] [Q&A]
I create vm with Sonoma, set my apple account all right.
After that I update by settings panel for Sequoia 15.2 my apple account no longer log in.
Show a message: An unknown error occured.
And not work more.

**Comment by wijzijnweb** [comment]
Same here, did you manage to solve it?

**Comment by Fernand117** [comment]
I have this problem, did you manage to solve?

**Comment by wperesdev** [comment]
I hope it helps you:

https://www.youtube.com/watch?v=PZXS3AztbmQ
